ipinfo/cmd_default.go

```diff
@@ -108,6 +110,19 @@ func cmdDefault() (err error) {
 		return nil
 	}
 
+	if fHelpDetailed {
+		// Read the string and display it using a pager
+		err = lib.HelpDetailed(DetailedHelp)
```

Comment:
Can we have this accept the `printHelpDefault` function as an argument and then have it called inside for the error case? Dont forget over time we'll be adding this logic everywhere and it'll just add more and more code to be copying these error cases everywhere on the outside level